src/models/Scholarships.ts

```diff
@@ -65,6 +69,12 @@ class Scholarships extends FirestoreCollection<ScholarshipData> {
     if (opts.authorId) {
       query = query.where('authorId', '==', opts.authorId);
     }
+
+    query = query.limit(5).startAfter(lastDoc || 0);
+
+    const snapShot = await query.get();
+    const last = snapShot.docs[snapShot.docs.length - 1];
+    setLastDoc(last);
     return FirestoreCollection.list(query);
```

Comment:
Maybe what we can do is return some sort of response that has the results as well as a function to get the next page of results. E.g. 

```js
return FirestoreCollection.list(query)
  .then((scholarships) => { scholarships, next: () => Scholarships.list(opts, lastDoc)})
```